packages/grant-explorer/src/features/api/voting.ts

```diff
@@ -232,3 +244,26 @@ export const signPermitDai = async ({
   ];
   return { r, s, v: hexToNumber(v) };
 };
+
+export function encodeQFVotes(
+  donationToken: PayoutToken,
+  donations: Pick<
+    CartProject,
+    "amount" | "recipient" | "projectRegistryId" | "applicationIndex"
+  >[]
+): Hex[] {
+  return donations.map((donation) => {
+    const vote = [
+      getAddress(donationToken.address) as Hex,
```

Comment:
https://github.com/gitcoinco/grants-stack/issues/2158

;)

Reply:
This cast is not actually even necessary here. Good catch.